src/cephadm/cephadm

```diff
@@ -1948,13 +1948,14 @@ def get_last_local_ceph_image(ctx: CephadmContext, container_path: str) -> Optio
                             [container_path, 'images',
                              '--filter', 'label=ceph=True',
                              '--filter', 'dangling=false',
+                             '--sort', 'created',
                              '--format', '{{.Repository}}@{{.Digest}}'])
     return _filter_last_local_ceph_image(out)
 
 
 def _filter_last_local_ceph_image(out):
     # type: (str) -> Optional[str]
-    for image in out.splitlines():
+    for image in reversed(out.splitlines()):
```

Comment:
Do we still want this reversal if we are already adding in sorting by created? E.g. with

```
[root@vm-00 ~]# podman images --filter label=ceph=True --filter dangling=false --sort created
REPOSITORY                 TAG         IMAGE ID      CREATED            SIZE
quay.ceph.io/ceph-ci/ceph  quincy      30ed3ebe4d35  37 minutes ago     1.32 GB
quay.ceph.io/ceph-ci/ceph  pacific     32da8bfe2583  About an hour ago  1.25 GB
```
we would want to select the quincy image as it was created more recently but if we reverse these results we'd get the pacific one.

Reply:
Actually "created" is the default option for `--sort` option (from podman images -h):

```
      -q, --quiet            Display only image IDs
      --sort string      Sort by created, id, repository, size, tag (default "created")

```

I was confused because on my cluster, as starting version I used octopus image from the **ceph-ci** which is in practice older than 16,2 but it's listed in the top!

```
[root@ceph-node-0 ~]# podman images --filter label=ceph=True --filter dangling=false --sort created
REPOSITORY                 TAG               IMAGE ID      CREATED       SIZE
quay.ceph.io/ceph-ci/ceph  octopus           bffe274290f3  4 days ago    1.21 GB
quay.io/ceph/ceph          v16.2.7-20220302  1d0ab362ee4c  4 days ago    1.25 GB
quay.io/ceph/ceph          v16.2.7-20220303  1d0ab362ee4c  4 days ago    1.25 GB
quay.io/ceph/ceph          v16.2             1d0ab362ee4c  4 days ago    1.25 GB

```

Reply:
I'm removing the 'reverse' function since current code is already picking the more recent image. I'm keeping the `--sort created` option to enforce using always the most recent ceph image.

Reply:
reversing the list actually selects the oldest, not newest image ..